test/e2e/networks/rotate.toml

```diff
@@ -24,21 +24,21 @@ validator03 = 100
 validator04 = 100
 validator05 = 100
 
-[validator_update.1010]
+[validator_update.1050]
```

Comment:
the previous height values are stable (checked locally)
if you don't have any concerns then let's rollback to them